xmpp_muc.go

```diff
@@ -90,7 +90,7 @@ func (c *Client) JoinProtectedMUC(jid, nick string, password string, history_typ
 	case NoHistory:
 		return fmt.Fprintf(c.conn, "<presence to='%s/%s'>\n" +
 			"<x xmlns='%s'>\n" +
-			"<password>%s</password>\n"+
+			"<password>%s</password></x>\n"+
```

Comment:
better to be

```
 			"<x xmlns='%s'>\n" +
			"<password>%s</password>\n" + 
			"</x>\n" +
```

Reply:
The formatting with newline or appended is mixed throughout the file :wink: